_includes/allsids.txt

```diff
@@ -8,9 +8,15 @@
       {% elsif post.vote-status == 'deferred' %}
         <span class="label warning">Deferred</span>
       {% elsif post.vote-status == 'postponed' %}
-        <span class="label warning">Postponed</span>
+        <span class="label light-warning">Postponed</span>
+      {% elsif post.vote-status == 'under review' %}
+        <span class="label warning">Under review</span>
+      {% elsif post.vote-status == 'numbered' %}
+        <span class="label notice">Numbered</span>
       {% elsif post.vote-status == 'not accepted' %}
-        <span class="label danger">Not Accepted</span>
+        <span class="label danger">Rejected</span>
```

Comment:
Euphemism is the wrong word. It is less harsh to say "not accepted" rather than "rejected." Both are accurate, one just has a less negative connotation, which after you spend a lot of time proposing a change only to find that it wasn't accepted, may be what one needs. Less negativity. Maybe we should go back to "not accepted," Jorge? It was a strong preference of many a few years ago to do it this way. And I would also lean on keeping "not accepted" personally, myself.